connect/src/com/telenor/connect/id/IdProvider.java

```diff
@@ -0,0 +1,18 @@
+package com.telenor.connect.id;
+
+public enum IdProvider {
+    CONNECT_ID("connect.telenordigital.com", "connect.staging.telenordigital.com"),
+    TELENOR_ID("example.com", "staging-telenorid.com"); // TODO: production is not set yet
```

Comment:
I think the decision is to not use toplevel domain for the IDP, but rather signin.staging-telenorid.com

Reply:
Yes, you are right Vidar!